.github/workflows/windows.yml

```diff
@@ -0,0 +1,33 @@
+name: windows
+on: [push, pull_request]
+jobs:
+  build:
+    name: >-
+      build (windows, ${{ matrix.ruby }})
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [ '2.4.x', '2.5.x', '2.6.x' ]
+    steps:
+      - name: git config
+        run: git config --global core.autocrlf false
+      - name: Checkout
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 10
+      - name: Load ruby, update MSYS2/MinGW
+        uses: MSP-Greg/actions-ruby@master
+        with:
+          ruby-version: ${{ matrix.ruby }}
+          base: update
+      - name: Clone MSpec
+        run:  git clone https://github.com/ruby/mspec.git ../mspec
+      - name: Run specs
+        env:
+          CHECK_LEAKS: true
+        run: |
+          # Actions uses UTF8, causes test failures, similar to normal OS setup
+          [Console]::OutputEncoding = [System.Text.Encoding]::GetEncoding("IBM437")
+          [Console]::InputEncoding  = [System.Text.Encoding]::GetEncoding("IBM437")
```

Comment:
I think we should just adapt those 4 specs if they don't pass on Windows otherwise.
I'm surprised the behavior differs on Windows though, is that a known bug?

Reply:
I'll look at it again, I did find some odd things with it testing locally.  I try to stay away from encoding issues, as I'm running a standard en-US setup.  I think (but I'm not sure) that when the above encodings were left as-is, that things were US-ASCII ?

Reply:
Exclude these specs in https://github.com/ruby/spec/pull/726